### ISSUE TITLE
Empty Network Stats

### ISSUE THREAD

**Issue by ragnayev** [z-classic/z-nomp#212]
Hi,

I started Zen and Hush and both with empty Network Stats.
Both wallets works fine, no errors. Z-NOMP pool with only warning
"No rewardRecipients have been setup". In debug Z-NOMP window I can see 
2017-10-21 18:51:51 [Pool]       [hush] (Thread 1) Stratum Pool Server Started for hush [HUSH] {equihash}
                                                Network Connected:      Mainnet
                                                Detected Reward Type:   POW
                                                Current Block Height:   191123
                                                Current Block Diff:     50598.003996625
                                                Current Connect Peers:  13
                                                Network Difficulty:     206473156.2148627
                                                Network Hash Rate:      2.70 MH
                                                Stratum Port(s):        3032
                                                Pool Fee Percent:       0%
                                                Block polling every:    500 ms
2017-10-21 18:51:51 [Switching]     [Setup] (equihash) Setting proxy difficulties after pool start

How to fix it?

Regards,
Tim

**Comment by rizzman** [comment]
Yup... I too am having the same issue. Everything else appears to be working except the network stats. One thing to note; I cloned the zcash.json under pool_configs and coins then changed everything referencing zcash to hush. with the exception of the mpos which is not enabled. is there a log somewheres I can look to find out more info?

Thanks 
Rizz

**Comment by ragnayev** [comment]
Investigating the code I found that "Network Stats" gathering only if you enable "paymentProcessing" in pool_configs/coiname.json

Be sure you have access to resource https://api.coinmarketcap.com/v1/ticker/{coin}/
{coin} - submit by coin name e.g. hush
All list of coins you can get from https://api.coinmarketcap.com/v1/ticker/

Regards
Tim
